doc/mgr/dashboard.rst

```diff
@@ -503,6 +504,38 @@ e.g. caused by certificates signed by unknown CA or not matching the host name::
 
 You can directly access Grafana Instance as well to monitor your cluster.
 
+Alternative URL for Browsers
+""""""""""""""""""""""""""""
```

Comment:
Shouldn't this be caret (^) instead of double-quotes (") ? @p-se

Reply:
It seems that for octopus this is true, although it isn't for the master branch. I'll have to check. Thanks for the hint!

Reply:
@nizamial09 This is a backport, so we would not make this change here. If you want this change, may I suggest to open a PR targeting master?